server/azureAppInsights.ts

```diff
@@ -64,5 +64,6 @@ export default function initialiseAppInsights(): void {
     // custom processors to fine tune behaviour
     defaultClient.addTelemetryProcessor(ignoreExcludedRequestsProcessor)
     defaultClient.addTelemetryProcessor(addUsernameProcessor)
+    defaultClient.addTelemetryProcessor(errorStatusCodeProcessor)
```

Comment:
What do you think about
```suggestion
    defaultClient.addTelemetryProcessor(markClientErrorsAsSuccessProcessor)
```

Reply:
left it as a generic status code processor for now, but added a comment which explains how the status code affects the success attribute of the event